### PR TITLE
fix: handle result to unknown when istio_version is missing

### PR DESCRIPTION
**Pull request by Xunzhuo**
**What this PR does / why we need it**:

fix: handle result to unknown when istio_version is missing

**Special notes for your reviewer**: cc @bianpengyuan

### CODE REVIEW

**Review by bianpengyuan — COMMENTED** [review]
lgtm, although likely there will be lint error for line too long.

File: extensions/stats/plugin.cc [hunk 1]
```diff
@@ -450,9 +450,9 @@ bool PluginRootContext::initializeDimensions(const json& j) {
   Metric build(MetricType::Gauge, absl::StrCat(stat_prefix, "build"),
                {MetricTag{"component", MetricTag::TagType::String},
                 MetricTag{"tag", MetricTag::TagType::String}});
+  std::string istio_version = flatbuffers::GetString(local_node.istio_version()) == "" ? absl::StrCat(unknown, ";") : absl::StrCat(istio_version, ";");
```
Comment (bianpengyuan) [inline extensions/stats/plugin.cc:453]: This seems wrong.. istio_version is used before it is defined.

**Review by bianpengyuan — CHANGES_REQUESTED** [review]

File: extensions/stats/plugin.cc (hunk 1 above)
Comment (Xunzhuo) [reply extensions/stats/plugin.cc]: @bianpengyuan Thanks Peter~ I think it`s ok now

**Comment by Xunzhuo** [comment]
cc @bianpengyuan  I think it`s ok now, should I rebase these commits?

**Review by bianpengyuan — APPROVED** [review]

**Pull request merged**